app/lib/steps/capital_section.rb

```diff
@@ -11,23 +11,26 @@ def all_steps
 
       def all_steps_for_current_feature_flags
         if FeatureFlags.enabled?(:household_section)
-          %i[vehicle vehicles_details assets].freeze
+          %i[assets partner_assets vehicle vehicles_details].freeze
         else
-          (PROPERTY_STEPS + %i[vehicle vehicle_details] + TAIL_STEPS).freeze
+          (PROPERTY_STEPS + %i[vehicle vehicle_details assets]).freeze
         end
       end
 
       def grouped_steps_for(session_data)
         return [] if Steps::Logic.asylum_supported?(session_data)
 
-        [property_steps(session_data),
-         vehicle_steps(session_data),
-         Steps::Group.new(*TAIL_STEPS)].compact
+        if FeatureFlags.enabled?(:household_section)
+          [Steps::Group.new(*TAIL_STEPS),
```

Comment:
I don't think we want these screens in the same Group - at the moment, if you change client assets from the check answers screen, you're taken to the partner assets screen next before you get back to the check answers screen, which seems weird to me. I think they each belong in their own group.

Reply:
Also, if the client has no partner, we don't want to include partner_assets at all.